src/pal/src/exception/seh-unwind.cpp

```diff
@@ -92,14 +92,14 @@ static void WinContextToUnwindContext(CONTEXT *winContext, unw_context_t *unwCon
 static void UnwindContextToWinContext(unw_cursor_t *cursor, CONTEXT *winContext)
 {
 #if defined(_AMD64_)
-    unw_get_reg(cursor, UNW_X86_64_RIP, &winContext->Rip);
-    unw_get_reg(cursor, UNW_X86_64_RSP, &winContext->Rsp);
-    unw_get_reg(cursor, UNW_X86_64_RBP, &winContext->Rbp);
-    unw_get_reg(cursor, UNW_X86_64_RBX, &winContext->Rbx);
-    unw_get_reg(cursor, UNW_X86_64_R12, &winContext->R12);
-    unw_get_reg(cursor, UNW_X86_64_R13, &winContext->R13);
-    unw_get_reg(cursor, UNW_X86_64_R14, &winContext->R14);
-    unw_get_reg(cursor, UNW_X86_64_R15, &winContext->R15);
+    unw_get_reg(cursor, UNW_REG_IP, (unw_word_t *) &winContext->Rip);
```

Comment:
Does this work on Linux? I don't see the UNW_REG_IP/SP in the libunwind.h, only the ones I've used.